feat: update english & portuguese translation

igormiguell opened the pull request.

xfqwdsj commented on app/src/main/res/values/strings.xml:
```diff
@@ -6,13 +6,13 @@
     <string name="status_normal">Working normally</string>
     <string name="status_partial">Partially working</string>
-    <string name="status_error">May not work</string>
+    <string name="status_error">It\'s not working</string>
```
Please keep this.

xfqwdsj commented on app/src/main/res/values/strings.xml:
```diff
@@ -6,13 +6,13 @@
     <string name="status_entry_prefs_y">Preferences ready</string>
     <string name="status_entry_prefs_n">Preferences not working</string>
-    <string name="status_entry_prefs_description">The state of Preferences determines whether the module settings can be saved. Generally, proper usage of LSPosed will not cause this state to be abnormal. If this state is abnormal, all module features will be enabled by default.</string>
+    <string name="status_entry_prefs_description">The state of a preference determines whether the module\'s settings can be saved. Normally, proper use of LSPosed won\'t cause this state to become abnormal. If the state is abnormal, all module features will be enabled by default.</string>
```
Please keep `Preferences`. #90 is an example.

xfqwdsj commented on app/src/main/res/values-pt-rBR/strings.xml:
```diff
@@ -1,8 +1,20 @@
+    <string name="status_entry_activation_description">O status de ativação do módulo afeta diretamente sua funcionalidade. Se o módulo não estiver ativado, ele não funcionará. O módulo relata seu status de ativação conectando seus próprios componentes. Se você suspeitar de um relatório falso, force a parada do app e abra-o novamente.</string>
+    <string name="status_entry_prefs_y">Preferências prontas</string>
+    <string name="status_entry_prefs_n">Preferências não funcionam</string>
+    <string name="status_entry_prefs_description">O estado de uma preferência determina se as configurações do módulo podem ser salvas. Normalmente, o uso adequado do LSPosed não torna esse estado anormal. Se o estado for anormal, todos os recursos do módulo serão ativados por padrão.</string>
```
Maybe this should be updated too.

xfqwdsj reviewed (changes requested):
I'm more curious as to why you would want to update the English strings. I'm also learning English, can you tell me what your considerations are for these changes?

igormiguell commented:
> I'm more curious as to why you would want to update the English strings. I'm also learning English, can you tell me what your considerations are for these changes?

Hi! I decided to change the English strings to a more informal style, following the writing conventions of the Android system. Since the app also follows the standard Android design, I thought it would be interesting to use this type of language, which gives the impression of a native system app.

xfqwdsj reviewed (changes requested):
Since I cannot read Portuguese, please double-check that the translation corresponds to the English one, especially the changes I requested earlier.

igormiguell commented:
> Since I cannot read Portuguese, please double-check that the translation corresponds to the English one, especially the changes I requested earlier.

Yes, everything is correct. Ready to merge.

xfqwdsj commented:
Nope, you haven't edit "may not" and "Preferences".

With regard to “may not”, this is a critical expression, as there may be various reasons for a false report.

With regard to “Preferences”, this is the name of a system feature and should be kept.

Why do I ask for this? As you mentioned, the style of this application is more formal, and I don't think there should be any deviation in expression in different languages, so please understand.

igormiguell commented:
> Nope, you haven't edit "may not" and "Preferences".
> 
> With regard to “may not”, this is a critical expression, as there may be various reasons for a false report.
> 
> With regard to “Preferences”, this is the name of a system feature and should be kept.
> 
> Why do I ask for this? As you mentioned, the style of this application is more formal, and I don't think there should be any deviation in expression in different languages, so please understand.

Sorry for the previous mistake. It's now fixed. I have added deepwiki to the README_PT-BR. If it's not necessary, just let me know and I'll remove it without any problem.

By the way, I have a question: "Preferences", as mentioned, is the name of a system feature, would it be more appropriate to change the string status_entry_prefs_y to "Preferences prontas"?

xfqwdsj commented:
> change the string status_entry_prefs_y to "Preferences prontas"

Maybe? [This document](https://developer.android.com/training/data-storage/shared-preferences?hl=pt_br) shows one usage, especially its title.

igormiguell commented:
You can go ahead and merge it, no problem.

xfqwdsj commented:
So the different translations of `Preferências` and `Preferences` in `status_entry_prefs_y`, `status_entry_prefs_n`, and `status_entry_prefs_description` are deliberate on your part? I need final confirmation on this.

igormiguell commented:
yes
